docs/learn/expert-guides/key-manager/execute-relay-transactions.md

```diff
@@ -8,91 +8,71 @@ import TabItem from '@theme/TabItem';
 
-## Generate the signed transaction payload
+## Setup
 
-This example shows how to prepare a transaction to be executed by a third party. This logic can be implemented client-side and then sent to a third-party application or service such as a Transaction Relay service to be executed.
+First, the transaction to be executed by a third party has to be prepared. This logic can be implemented:
 
-Make sure you have the following dependencies installed before beginning this tutorial:
+- **client-side** and sent to a third-party application
+- **service-side**, if the Transaction Relay Service is set up on the Universal Profile
 
-- Either [`web3.js`](https://github.com/web3/web3.js) or [`ethers.js`](https://github.com/ethers-io/ethers.js/)
-- [`@lukso/lsp-smart-contracts`](https://github.com/lukso-network/lsp-smart-contracts/)
-- [`@lukso/eip191-signer.js`](https://github.com/lukso-network/tools-eip191-signer)
+You will need the following dependencies installed:
 
-<Tabs>
-  
-  <TabItem value="web3js" label="web3.js">
+- a blockchain provider library like [`ethers`](https://github.com/ethers-io/ethers.js/) or [`web3`](https://github.com/web3/web3.js)
+- the [`@lukso/lsp-smart-contracts`](https://github.com/lukso-network/lsp-smart-contracts/) package to get the artifacts of the [LSP contracts](../../../standards/introduction.md)
+- the [`@lukso/eip191-signer.js`](https://github.com/lukso-network/tools-eip191-signer) library to sign the relay transaction
 
-```shell title="Install the dependencies"
-npm install web3 @lukso/lsp-smart-contracts @lukso/eip191-signer.js
+<Tabs groupId="provider-lib">
+
+  <TabItem value="ethers" label="ethers">
+
+```shell
+npm install ethers @lukso/lsp-smart-contracts @lukso/eip191-signer.js
 ```
 
   </TabItem>
 
-  <TabItem value="ethersjs" label="ethers.js">
+  <TabItem value="web3" label="web3">
 
-```shell title="Install the dependencies"
-npm install ethers @lukso/lsp-smart-contracts @lukso/eip191-signer.js
+```shell
+npm install web3 @lukso/lsp-smart-contracts @lukso/eip191-signer.js
 ```
 
   </TabItem>
 
 </Tabs>
 
-### Step 1 - Setup imports and constants
-
-To encode a transaction, we need the address of the Universal Profile smart contract and the private key of a controller key with sufficient [LSP6 permissions](../../../standards/universal-profile/lsp6-key-manager.md#permissions) to execute the transaction.
-
-<Tabs>
-  
-  <TabItem value="web3js" label="web3.js">
+:::info Execution Rights
 
-```typescript title="Imports & Constants"
-import UniversalProfileContract from '@lukso/lsp-smart-contracts/artifacts/UniversalProfile.json';
-import KeyManagerContract from '@lukso/lsp-smart-contracts/artifacts/LSP6KeyManager.json';
-import { EIP191Signer } from '@lukso/eip191-signer.js';
-import Web3 from 'web3';
+Even when relaying the transaction using a third-party, you will need execute permission is necessary. Otherwise, unauthorized parties would be able to send transactions to your relayer address.
```

Comment:
Suggest adding link to the permission in the LSP6 standard page. And remove the 2nd sentence which contradicts the first.

```suggestion
To execute relay transactions using a third-party add, you will need the `EXECUTE_RELAY_CALL` permission.
```